src/js/components/Anchor/__tests__/__snapshots__/Anchor-test.js.snap

```diff
@@ -60,6 +60,7 @@ exports[`Anchor focus renders 1`] = `
   text-decoration: none;
   cursor: pointer;
   outline: none;
+  outline-color: #6FFFB0;
```

Comment:
Are changes on Anchor component and Button are expected with this PR?

Reply:
Yes. I added the `outline-color` focus style as that seems to work better for tbody.